vm/isolated/isolated.go

```diff
@@ -65,6 +71,7 @@ func ctor(env *vmimpl.Env) (vmimpl.Pool, error) {
 	if env.Debug && len(cfg.Targets) > 1 {
 		log.Logf(0, "limiting number of targets from %v to 1 in debug mode", len(cfg.Targets))
 		cfg.Targets = cfg.Targets[:1]
+		cfg.USBDevNum = cfg.USBDevNum[:1]
```

Comment:
Does this work when `USBDevNum` is empty? I think we need a check for its length like for `Targets`.